Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/organizer/details/QrCodeDetailFragment.kt

```diff
@@ -152,6 +159,20 @@ class QrCodeDetailFragment : Fragment(R.layout.trace_location_organizer_qr_code_
         behavior.overlayTop = ((width) / 2) - 24
     }
 
+    private fun openCreateEventFragment(traceLocation: TraceLocation) {
+        val category = traceLocationCategories.find { it.type == traceLocation.type }
+        if (category == null) {
+            Timber.e("Category not found, traceLocation = $traceLocation")
+        } else {
+            findNavController().navigate(
```

Comment:
Please consider having the same transition as `To QrCodePosterFragment` for consistency?

Reply:
updated